.github/workflows/test_frameworks.yml

```diff
@@ -634,7 +670,7 @@ jobs:
           pip install ../ddtrace
       - name: Install dependencies
         if: needs.needs-run.outputs.outcome == 'success'
-        run: scripts/install
```

Comment:
NB: Was this a script that got deleted? Couldn't find it 😅

Reply:
This is the script: https://github.com/encode/starlette/blob/master/scripts/install

I removed that call because upgrading `pip` causes some failures, example:
https://github.com/DataDog/dd-trace-py/actions/runs/9662999797/job/26654192418#step:6:21

Reply:
I'm trying to find out the root cause for those failures, but seems to be the latest `pip` release